src/main/java/frc/robot/subsystems/vision/PhotonSubsystem.java

```diff
@@ -22,9 +22,9 @@
 import org.photonvision.targeting.PhotonPipelineResult;
 import org.photonvision.targeting.PhotonTrackedTarget;
 
-public class PhotonSubsystem extends SubsystemBase implements VisionSubsystem {
-  private int measurementsUsed = 0;
-  private DebugEntry<Integer> measurementEntry = new DebugEntry<Integer>(0, "measurements", this);
+public class PhotonSubsystem extends VisionSubsystem {
+  private double measurementsUsed = 0;
```

Comment:
Why was this changed to a double? Iterators are usually ints. In some languages float types _can't_ be iterators.

Reply:
The DebugEntry class doesn't have functionality to accept Integers so I replaced it with a Double, it should work the same I believe